src/StreamrClient.js

```diff
@@ -28,18 +29,15 @@ export default class StreamrClient extends EventEmitter {
             autoConnect: true,
             // Automatically disconnect on last unsubscribe
             autoDisconnect: true,
-            apiKey: null,
+            auth: {},
         }
         this.subsByStream = {}
         this.subById = {}
         this.publishQueue = []
 
         Object.assign(this.options, options || {})
 
-        // Backwards compatibility for option 'authKey' => 'apiKey'
-        if (this.options.authKey && !this.options.apiKey) {
-            this.options.apiKey = this.options.authKey
```

Comment:
Whoops, was the backwards compatibility now accidentally dropped? Please add two test cases to ensure it works:

1. test `new StreamrClient({ authKey: 'foo' })` results in `this.options.auth.apiKey === 'foo'`
2. test `new StreamrClient({ apiKey: 'foo' })` results in `this.options.auth.apiKey === 'foo'`